app/controllers/adoptions_controller.rb

```diff
@@ -2,21 +2,17 @@ class AdoptionsController < ApplicationController
   include SessionVerifiable
 
   before_action :find_rubygem
-  before_action :verify_ownership_requestable
   before_action :redirect_to_verify, if: -> { current_user_is_owner? && !verified_session_active? }
 
   def index
+    authorize @rubygem, :show_adoption?
```

Comment:
in other places you've done this by overriding `find_rubygem`, can we be consistent?

Reply:
good point, I'll converge towards the override approach for now, and we can refactor if necessary.